scripts/test.sh

```diff
@@ -128,14 +128,6 @@ function integration_pass {
   integration_extra "$@"
 }
 
-function e2e_pass {
-  # e2e tests are running pre-build binary. Settings like --race,-cover,-cpu does not have any impact.
-  # shellcheck disable=SC2068
-  run_for_module "tests" go_test "./e2e/..." "keep_going" : -timeout="${TIMEOUT:-30m}" ${RUN_ARG[@]:-} "$@" || return $?
-  # shellcheck disable=SC2068
-  run_for_module "tests" go_test "./common/..." "keep_going" : --tags=e2e -timeout="${TIMEOUT:-30m}" ${RUN_ARG[@]:-} "$@"
-}
-
```

Comment:
Hmm...I am looking into it. I think this function is never called in the tests triggered from the makefile. I wll verify this. Also, I think this can be moved into the test_lib.sh and defined there.

Reply:
Implementing a few fixes. Will update soon.

Reply:
@jmhbnz I have moved some of the functions to the `test_lib.sh` script. That way the other scripts will be simple. We can split up the `test_lib.sh` script itself later